distribution/lib/Standard/Table/0.0.0-dev/src/Data/Table.enso

```diff
@@ -1308,45 +1309,69 @@ type Table
     write_json : File.File -> Nothing
     write_json file = this.to_json.to_text.write file
 
-    ## UNSTABLE
+    ## This function writes a table from memory into a file.
 
-       Writes the table to a specified file with the given serialization
-       settings.
+       The specific behavior of the various `File_Format`s is specified below.
 
        Arguments:
-       - file: the file to write to.
-       - format: the format settings to use.
-
+       - path: The path to the output file.
+       - format: The format of the file.
+         If `File_Format.Auto` is specified; the file extension determines the
+         specific type and uses the default settings for that type to be used.
+         Details of this type are below.
+       - on_existing_file: Specified how to handle if the file already exists.
+       - column_matching: Specifies how to map columns against an existing file.
+         If `Column_Matching.By_Name` - the columns are mapped by name against
+         an existing file.
+         If `Column_Matching.By_Position` - the columns are mapped by position
+         against an existing file.
+         If there is a mismatch, then a `Column_Mismatch` error is raised.
+       - on_problems: Specifies how to handle if a problem occurs, raising as a
+         warning by default. The specific issues depend on the `File_Format`
+         argument.
+
+       Returns:
+       - If an unsupported `File_Format` is specified, an
+         `Illegal_Argument_Error` is raised.
+       - If the path to the parent location cannot be found or the filename is
+         invalid, a `File_Not_Found` is raised.
+       - If another error occurs, such as access denied, an `Io_Error` is
```

Comment:
The wording is slightly confusing. It says "If another error occurs" but later you list other possible exceptions so I'm guessing this should be something like "If another IO error occurs" ?